charts/spire/values.yaml

```diff
@@ -7,6 +7,8 @@ global:
     trustDomain: ""
     # -- Override all instances of bundleConfigMap
     bundleConfigMap: ""
+    # -- This is the value of your clusters `kubeadm init --service-dns-domain` flag
+    clusterDomain: ""
```

Comment:
This is a non spire var.

Lift one level up.

    global.clusterDomain

Reply:
And clusterDomain, while makes sense in the context of spire, could mean different things in the context of a chart depending on the spire chart. Its not very specific a value.